Dockerfile.repo

```diff
@@ -46,9 +48,9 @@ RUN \
 
 RUN \
   cd ~ && \
-  wget https://github.com/jeffreyhorner/rapache/archive/v1.2.9.tar.gz -O rapache-1.2.9.tar.gz && \
-  tar xzvf rapache-1.2.9.tar.gz && \
-  cd rapache-1.2.9 && \
+  wget https://github.com/jeffreyhorner/rapache/archive/master.tar.gz -O rapache-master.tar.gz && \
```

Comment:
Rapache was failing to compiles for arm64 https://github.com/jeffreyhorner/rapache/issues/57 so I created a fix and got it into the rapache repo via jeffreyhorner/rapache#58 there's no official release so went to master.  This is just for the r-repo which doesn't change often so I think its ok to not lock it.

Reply:
nit: set an RAPACHE_VERSION variable with current value `master`.